### ISSUE TITLE
uncompatibility with animate.css library

### ISSUE THREAD

**Issue by Loque18** [teodosii/react-notifications-component#155]
if you import the animate.css into the project, the first time you dispatch a notification if dissapears for a second, then it appears again, (at least trying to play the fade in animation) 

**stepts to reproduce:** 

create new react / nextjs app
npm install react-notifications-component@4.0.0
npm install animate.css

import react-notifications-component
import animate.css

dispatch a notification with fade in entrance animation

**Comment by teodosii** [comment]
Let's stick to one issue open for now :)

**Comment by djizco** [comment]
I started having this or a similar issue when I migrated to React 18. First time dispatching a notification, it appears for a millisecond, then disappears, and then does the animation.

**Comment by Loque18** [comment]
> I started having this or a similar issue when I migrated to React 18. First time dispatching a notification, it appears for a millisecond, then disappears, and then does the animation.

Yes I still having this issue, the only solution I found for now is to stop using animate.css

**Comment by teodosii** [comment]
For some reason I thought the reporter reported the same issue twice, when the other one was fixed in a different thread but it seems it's not the same. I'll investigate, will let it open for now.

**Comment by teodosii** [comment]
> I started having this or a similar issue when I migrated to React 18. First time dispatching a notification, it appears for a millisecond, then disappears, and then does the animation.

Interesting, this doesn't seem to happen on `github-pages` - Are you able to replicate there?

**Comment by Yekna** [comment]
I was also able to replicate it in a React 18 project.
